lib/pages/home.dart

```diff
@@ -1,170 +1,170 @@
-
-    return new Padding(
-      padding: const EdgeInsets.symmetric(vertical: 10.0),
-      child: new Center(
-        child: new Column(
-          children: <Widget>[
-            new CircleAvatar(
-              backgroundImage: new NetworkImage(
-                  'https://avatars.githubusercontent.com/${model.username}'),
-            ),
-            new Text(
-              model.username,
-              style: new TextStyle(
-                fontSize: 20.0,
-              ),
-            ),
-            value > 0
-                ? new Text(
-                    'Today\'s contribution is ${value}! 👍',
-                    style: new TextStyle(
-                      color: Colors.green,
-                      fontWeight: FontWeight.bold,
-                    ),
-                  )
-                : new Text(
-                    '${todayLabel} is not contributed yet.❌',
-                    style: new TextStyle(
-                      color: Colors.red,
-                    ),
-                  ),
-          ],
-        ),
-      ),
-    );
+  class _ContributionTile {
+    final int year;
+    final int month;
+    final String yearMonth;
+    final List<int> contributions;
+
+    _ContributionTile({this.year, this.month})
+        : contributions = [],
+          yearMonth = '${year.toString()}-${month.toString().padLeft(2,'0')}';
+
+    bool isPresentMonth() {
+      DateTime today = new DateTime.now();
+      return (year == today.year && month == today.month);
+    }
+
+    @override
+    String toString() => '${yearMonth}:' + contributions.length.toString();
   }
 
-  Widget _createMonthlyContributionMap(_ContributionTile monthlyContrib) {
-    return new Column(
-      children: <Widget>[
-        new Text(monthlyContrib.yearMonth),
-        new Row(
-          mainAxisSize: MainAxisSize.max,
-          children: monthlyContrib.contributions.map(
-            (c) {
-              Widget valueText = new Text(
-                c.toString(),
+  class _MyHomePageState extends State<_InnerMyHomePage> {
+    final GitHubUser model;
+    List<_ContributionTile> monthlyContributions = [];
+
+    _MyHomePageState(this.model) {
+      monthlyContributions = model.contributions
+          .fold(<_ContributionTile>[],
+              (List<_ContributionTile> list, Contribution contib) {
+            String key =
+                '${contib.year.toString()}-${contib.month.toString().padLeft(2,'0')}';
+            _ContributionTile tile;
+            if (!list.any((c) => c.yearMonth == key)) {
+              tile =
+                  new _ContributionTile(year: contib.year, month: contib.month);
+              list.add(tile);
+            } else {
+              tile = list.firstWhere((c) => c.yearMonth == key);
+            }
+            tile.contributions.add(contib.value);
+            return list;
+          })
+          .reversed
+          .toList();
+    }
+
+    @override
+    Widget build(BuildContext context) {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Use => for short members whose body is a single return statement.